plugins/woocommerce/src/Blocks/BlockTypes/ProductFilterClearButton.php

```diff
@@ -43,13 +43,19 @@ protected function get_block_type_script( $key = null ) {
 	 */
 	protected function render( $attributes, $content, $block ) {
 		// don't render if its admin, or ajax in progress.
-		if ( is_admin() || wp_doing_ajax() ) {
+		if (
+			is_admin() ||
```

Comment:
Have you actually tested if the calls to `is_admin` and `wp_doing_ajax`? From what I remember in the past, they don't work. It's always null even in the admin/editor.